nr-find-log4j.js

```diff
@@ -429,7 +429,7 @@ function writeResults(state) {
     state.scanDurationSec = Math.ceil((state.scanCompleted - state.scanStarted) / 1000);
 
     process.stdout.write(`\nOK, scan took ${state.scanDurationSec} seconds. Found ${vulnerableApplications.length} services with log4j-core.\n`);
-    const fileTimestamp = (new Date()).toISOString();
+    const fileTimestamp = new Date().toISOString().replaceAll(":","_");
```

Comment:
string.replaceAll() isn't supported in the Node v12 and v14 versions I tested with. we could use the regex global version of replace instead: `.replace(//g, '')`

Reply:
You're absolutely correct, apologies @fightingmonk - I had upgraded to 16 to see if it fixed the initial issue and didn't test backwards compatibility. New change suggested.